Altis_Life.Altis/dialog/spawnSelection.hpp

```diff
@@ -47,6 +47,7 @@ class life_spawn_selection {
             maxSatelliteAlpha = 0.75;//0.75;
             alphaFadeStartScale = 1.15;//0.15;
             alphaFadeEndScale = 1.29;//0.29;
+            widthRailWay = 4;
```

Comment:
As we're inheriting from Life_RscMapControl I don't see the need to repeat this, otherwise thanks for the PR.